### ISSUE TITLE
add error / success messages to 3dstreet-editor

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet-editor#224]
success messages:
- successful load of json, successful load of streetmix

error messages:
- streetmix or json loading error

**Comment by Algorush** [comment]
Message should appear after all elements from JSON are loaded? I am now thinking how to implement this, because the entity's 'loaded' event is used to load attributes/mixin of each element. There is such an option: when saving JSON, we can save the number of elements, and when loading from JSON, we can count how many elements have been loaded (in 'loaded' event) and show a message after all have been loaded. But it is not clear in which case to show the error message. If use try...catch...finally, then callbacks from the 'loaded' event are not taken into account, but we can use the catch to print a message if there was an error

**Comment by kfarr** [comment]
@Algorush I agree it is hard to decide what is the right way. I think at first successful load is: was the file loaded from http request or upload and was it parsed without an error (ie it was valid json)? Then whether or not it loads in a-frame is not actually monitored yet. We could do that later if we get reports that things aren't loading correctly.

But yes I think we should catch errors usually caused by not able to load the file (not getting HTTP response code 200) or the file itself is not parsable for some reason (invalid json).

Does that help?

**Comment by kfarr** [comment]
But even though this ticket is in `3dstreet-editor`, these messages should come from the json-utils I think and the specific task in this repo is to make sure the notification component is used in this project

**Comment by Algorush** [comment]
> But even though this ticket is in `3dstreet-editor`, these messages should come from the json-utils I think and the specific task in this repo is to make sure the notification component is used in this project

but JSON parsing and request/upload is not happening in json-utils.

**Comment by Algorush** [comment]
there is one more point. If displaing an upload message after a successful parse or request/upload, the message will appear before the scene has loaded.
I almost made the output of the message only after all the elements of the scene have loaded.  As well as displaying an error message at any stage of the loading scene. 
Maybe also make: if not all elements have loaded, then display a message: 423 out of 424 elements have loaded.

**Comment by Algorush** [comment]
https://github.com/3DStreet/3dstreet-editor/pull/240
spent a lot of time debugging and testing

**Comment by kfarr** [comment]
closed with #254 moved the "deep validation" of open json to a new ticket https://github.com/3DStreet/3dstreet/issues/334
